examples/resnet50/ImageNet Pretrained Network (ResNet-50).ipynb

```diff
@@ -10,6 +10,13 @@
     "\n",
     "We will create a set of Lasagne layers corresponding to the Caffe model specification (prototxt), then copy the parameters from the caffemodel file into our model (like <a href=\"https://github.com/Lasagne/Recipes/blob/master/examples/Using%20a%20Caffe%20Pretrained%20Network%20-%20CIFAR10.ipynb\">here</a>).\n",
     "\n",
+    "This notebook produce *resnet50.pkl* file, which contains dictionary with following foelds:\n",
+    " * values: numpy array with parameters of the model\n",
+    " * synset_words: labels of classes\n",
+    " * mean_image: mean image which should be subtracted from each input image\n",
+    "\n",
+    "This file can be used for initialization of weights of the model created by *modelzoo/resnet50.py*.\n",
+    "\n",
     "## Download the model\n",
     "https://s3.amazonaws.com/lasagne/recipes/pretrained/imagenet/resnet50.pkl\n",
     "\n",
```

Comment:
Good introduction! The download links still suggests that the pkl file is needed to follow the notebook -- shouldn't you just remove the **Download the model** section?